tests/test_rfn_query.py

```diff
@@ -0,0 +1,138 @@
+import datetime as dt
+import pandas as pd
+from pandas.compat import lrange
+from pandas import DataFrame
+
+from bgbb.sql.sql_utils import run_rec_freq_spk, to_s3_fmt
+import pyspark
+from pytest import fixture
+
+
+spark = pyspark.sql.SparkSession.builder.appName("test").getOrCreate()
```

Comment:
This is being used in two places. I'd make a shared fixture in a [`conftest.py`](https://docs.pytest.org/en/latest/fixture.html?highlight=conftest#conftest-py-sharing-fixture-functions).